scripts/LedSignMock.py

```diff
@@ -0,0 +1,36 @@
+
+import curses
+import argparse
+
+parser = argparse.ArgumentParser(description="Prints mock output.")
+parser.add_argument('--mock-output', help = "Displays Mock Output", )
+args = parser.parse_args()
+
+menu = ['   ','SCE LED Sign Visualization ', ' ','Most Recent Health Check: ', 'Most Recent Sign Update: ',' ',
+'Current Sign Text:', 'Current Brightness:', 'Current Scroll Speed:', 'Current Text Color: ', 'Current Background:',
+'Current Border Color: ' ]
+
+class LedSignMock(data):
+    def print_menu(stdscr, selected_row_idx):
```

Comment:
if this is a class method, the first parameter needs to be `self`. [this explanation](https://www.geeksforgeeks.org/self-in-python-class/#:~:text=self%20represents%20the%20instance%20of,attributes%20with%20the%20given%20arguments.&text=self%20is%20parameter%20in%20function,name%20in%20place%20of%20it.) may help

```suggestion
    def print_menu(self, stdscr, selected_row_idx):
```